src/debug.cc

```diff
@@ -256,6 +256,9 @@ class SyslogChannel: public DebugChannel
 public:
     SyslogChannel(): DebugChannel("syslog") {}
 
+    /// whether log() ought to write the corresponding debugs() message
+    bool shouldWrite(const DebugMessageHeader &) const;
```

Comment:
```suggestion
    /// (assuming some higher-level code applied cache.log section/level filter)
    bool shouldWrite(const DebugMessageHeader &) const;
```

Or we should remove that caveat from the other shouldWrite() declaration.